.gitignore

```diff
@@ -19,3 +19,7 @@ yalc.lock
 
 build/
 !packages/ckeditor5-build-*/build
+
+# Ignore compiled TypeScript files.
+packages/*/src/**/*.js
```

Comment:
I'd keep packages/ckeditor5-utils/src/**/*.js only. This could interfere with work on other packages